Copr builds are broken (again)

### What Operating System(s) are you seeing this problem on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

Since november 2025 all build in Copr (https://copr.fedorainfracloud.org/coprs/wezfurlong/wezterm-nightly/) are broken.
The last know good is 20251111_071056_118802c2

### WezTerm version

20251111_071056_118802c2

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

No, and I'll explain why below

### Describe the bug

Can't download the latest nigtly for RPM based distros.

### To Reproduce

_No response_

### Configuration

no config

### Expected Behavior

_No response_

### Logs

_No response_

### Anything else?

_No response_

FTR, here are the failing build logs:
https://download.copr.fedorainfracloud.org/results/wezfurlong/wezterm-nightly/srpm-builds/09966264/builder-live.log.gz

Error occurs around here:
```
[...]
++ cat
+ BUILD_REQUIRES='BuildRequires: gcc, gcc-c++, make, curl, fontconfig-devel, openssl-devel, libxcb-devel, libxkbcommon-devel, libxkbcommon-x11-devel, wayland-devel, xcb-util-devel, xcb-util-keysyms-devel, xcb-util-image-devel, xcb-util-wm-devel, git
%if 0%{?suse_version}
BuildRequires: Mesa-libEGL-devel
%else
BuildRequires: mesa-libEGL-devel
%endif
%if 0%{?fedora} >= 41
BuildRequires: openssl-devel-engine
%endif
Source0: wezterm-${TAR_NAME}.tar.gz'
+ cat
+ test -n /mnt/safe-resultdir-bnj9u76n
+ /usr/bin/rpmbuild -bs --rmspec wezterm.spec --verbose
setting SOURCE_DATE_EPOCH=1696204800

RPM build errors:
error: Bad file: /builddir/build/SOURCES/wezterm-${TAR_NAME}.tar.gz: No such file or directory
    Bad file: /builddir/build/SOURCES/wezterm-${TAR_NAME}.tar.gz: No such file or directory
make: *** [/mnt/workdir-wfythdl7/wezterm/.copr/Makefile:2: srpm] Error 1
[...]
```

I don't know much about copr nor rpm builds to go further at this point.

--- a/ci/deploy.sh
+++ b/ci/deploy.sh
@@ -160,7 +160,7 @@ cargo build --release \
       -p strip-ansi-escapes
 BUILDEOFEOF
 )
-          BUILD_REQUIRES=$(cat <<'BREQEOF'
+          BUILD_REQUIRES=$(cat <<BREQEOF
 BuildRequires: gcc, gcc-c++, make, curl, fontconfig-devel, openssl-devel, libxcb-devel, libxkbcommon-devel, libxkbcommon-x11-devel, wayland-devel, xcb-util-devel, xcb-util-keysyms-devel, xcb-util-image-devel, xcb-util-wm-devel, git
 %if 0%{?suse_version}
 BuildRequires: Mesa-libEGL-devel

